exercises/simple-linked-list/.meta/hints.md

```diff
@@ -0,0 +1,35 @@
+## Implementation Hints
+
+Do not use the struct `SimpleLinkedList` just as a wrapper around a `Vec`, but use the fields `head` and `len`.  
+This might look similar to this:
+```
+pub struct SimpleLinkedList<T> {
+    head: Option<Box<Node<T>>>,
+    len: usize,
+}
+```
+The `head` field points to the first element (Node) of this linked list.  
+Also implements this struct `Node` with the following fields:
+```
+struct Node<T> {
+    data: T,
+    next: Option<Box<Node<T>>>,
+}
+```
+`data` contains the stored data, and `next` points to the following node (if available) or None.  
+
+#### Why Option<__Box__<Node<T>>> and not just Option<Node<T>>?
```

Comment:
If this is contained in a level 2 heading (`##`), it should be a level 3 heading (`###`).